packages/core/src/BlockLink/BlockLink.spec.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-import { BlockLink } from '..'
+import { BlockLink } from './BlockLink'
```

Comment:
I think we're trying to avoid imports from barrel files, so we may need to figure out a strategy for handling coverage reductions that's more targeted than excluding all `index.*` from the report.

Reply:
i think `src/**/index.ts` or similar should be fine. after this PR, all they do is export the component and the type